### PR TITLE
Added test that checks example bots' training data

### PR DESCRIPTION
**Pull request by degiz**
**Proposed changes**:

- Added unit test
- Fixed training data
- Fixed schema file for stories

**Status (please check what you already did)**:
- [x] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: tests/examples/test_example_bots_training_data.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+def get_config_file(bot_name: Text) -> Text:
+    return str(Path(EXAMPLES_FOLDER_ROOT) / bot_name / CONFIG_FILE)
+
+
+def get_domain_file(bot_name: Text) -> Text:
+    return str(Path(EXAMPLES_FOLDER_ROOT) / bot_name / DOMAIN_FILE)
+
+
+@pytest.mark.parametrize(
+    "bot_name", ["concertbot", "knowledgebasebot", "moodbot", "reminderbot", "rules"]
```
Comment (tmbo) [inline tests/examples/test_example_bots_training_data.py:28]: OPT: as much as I like your DRY approach, this breaks IDEs "find where this is used" feature as well as manual searches - I think in terms of keeping things simple there is no harm in just listing the paths here

Comment (degiz) [reply tests/examples/test_example_bots_training_data.py:28]: Do you mean listing explicitly:
- `examples/concertbot`
- `examples/knowledgebasebot`

etc?

Comment (degiz) [reply tests/examples/test_example_bots_training_data.py:28]: Not sure how to not break "find usage" anyway 😕

Comment (tmbo) [reply tests/examples/test_example_bots_training_data.py:28]: yes, I'd just list the files, e.g.
```
@pytest.mark.parametrize(
 "domain, config, ...", [
  ("examples/concertbot/domain.yml", "examples/concertbot/config.yml"),
  ("examples/knowledgebasebot/domain.yml","examples/knowledgebasebot/config.yml"),
  ...
```

**Pull request merged**